### ISSUE TITLE
Better exception handling

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Khang-NT/aacdecoder-android#31]
```
When playing some internet audio stream I got this error: 

java.lang.IllegalArgumentException: 88200Hz is not a supported sample rate.
        at android.media.AudioTrack.audioParamCheck(AudioTrack.java:369)
        at android.media.AudioTrack.<init>(AudioTrack.java:312)
        at android.media.AudioTrack.<init>(AudioTrack.java:265)
        at com.spoledge.aacdecoder.PCMFeed.run(PCMFeed.java:277)
        at java.lang.Thread.run(Thread.java:1019)

I know that this sample rate is bad but I think that this exception should be 
caught. Unfortunately it is not and application is crashing because of it :(

```

Original issue reported on code.google.com by `radoslaw...@gmail.com` on 2 Jan 2013 at 9:26

**Comment by GoogleCodeExporter** [comment]
```
And another example:

java.lang.NullPointerException
at com.spoledge.aacdecoder.BufferReader.run(BufferReader.java:155)
at java.lang.Thread.run(Thread.java:1096)


```

Original comment by `radoslaw...@gmail.com` on 19 Jan 2013 at 4:36

**Comment by GoogleCodeExporter** [comment]
```
Revision 21 - fixed the code in PCMFeed.

As for the second problem - I would like to know when this happened - this 
should not happen at all.
```

Original comment by `vbarta...@gmail.com` on 30 May 2013 at 9:50
- Changed state: **Started**
- Added labels: **Component-Java**

**Comment by GoogleCodeExporter** [comment]
```
Revision 27 (not 21)
```

Original comment by `vbarta...@gmail.com` on 12 Jan 2014 at 6:07

**Comment by GoogleCodeExporter** [comment]
```
fixed in version 0.7
```

Original comment by `vbarta...@gmail.com` on 12 Jan 2014 at 6:08
- Changed state: **Fixed**
